src/webapp/default_settings.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+
+"""
+Flask environment configuration
+"""
+
+from environs import Env
+
+env = Env()
+env.read_env()
+
+ENV = env.str("FLASK_ENV", default="production")
+DEBUG = True
+
+# For local development purposes
+SESSION_TYPE = "filesystem"
+
+# Azure AD configuration - Update these three to your values
+
+# This application's "Application (client) ID" found in this application's
+# app registration in Azure Active Directory
+# Example: "8a193f2d-e89d-c577-81b7-145f76344a77"
+CLIENT_ID = ""
```

Comment:
See if you can fit the comments on one line and then include all three in README snippet.

Reply:
When you say "fit the comments on one line" do you mean something like:

```python
CLIENT_ID = "" # Comment Here
AUTHORITY = "" # Comment Here
CLIENT_CREDENTIAL = "" # Comment Here
```

Or did you mean instead 

```python
# Only a single line comment here for all three.
CLIENT_ID = ""
AUTHORITY = ""
CLIENT_CREDENTIAL = ""
```

Or something else?

Reply:
Let me know if the new README + config + code layout works for you now in this regard.

Reply:
This is more what I had in mind (though need something better for the LOLWUT line if we can't use the two vars like other MSALs do):

```Python
# 'Application (client) ID' of app registration in Azure portal - this value is a GUID
CLIENT_ID = ""

# Client secret 'Value' (not its ID) from 'Client secrets' in app registration in Azure portal
CLIENT_CREDENTIAL = ""

# [LOLWUT] Concatenation of the identity provider instance, sign-in audience, and possibly the tenant ID
AUTHORITY = ""
```

Reply:
Ah, okay, that's what you mean by "one line" -- those are longer than the allowed `black` & `pylint` default formatting rules, but as they are comments and in a config file, I'll let them be, otherwise they would end up on multiple lines.

Addressed in https://github.com/Azure-Samples/ms-identity-docs-code-python/pull/4/commits/760ab64e273668380ed27b1b29c5051b87244d98